Xpotify/Scripts/Common/mouseWheelListener.ts

```diff
@@ -0,0 +1,24 @@
+﻿/// <reference path="statusReport.ts" />
+/// <reference path="action.ts" />
+
+namespace XpotifyScript.Common.MouseWheelListener {
+
+    declare var Xpotify: any;
+   
+    function volumeMouseWheelHandler(e) {
+        var { deltaY } = e;
+        var currentVolume = StatusReport.getVolume();
+        var newVolume = Math.max(Math.min(currentVolume - (deltaY / 1000), 1), 0);
+
+        Action.seekVolume(newVolume);
+    }
+
+    function setVolumeBarListener() {
+        var volumeBar = document.querySelector("#main > div > div.Root__top-container > div.Root__now-playing-bar > footer > div.now-playing-bar > div.now-playing-bar__right > div > div > div.volume-bar > div");
```

Comment:
This selector is too strict, and so it's prone to breaking by changes in PWA. Please use a more loose selector such as `.Root__top-container .Root__now-playing-bar .now-playing-bar__right__inner .volume-bar > div` for this.